cmd/libs/go2idl/client-gen/generators/generator-for-type.go

```diff
@@ -49,6 +50,16 @@ func (g *genClientForType) Imports(c *generator.Context) (imports []string) {
 	return g.imports.ImportLines()
 }
 
+// hasStatus returns if the type has a status field
+func hasStatus(t *types.Type) bool {
```

Comment:
Add a note/todo saying ideally we want to return true if the resource has a /status subresource.
But we do not have that information so for now, we return true if the type has a status field.